src/EnergyPlus/OutputReportTabularAnnual.cc

```diff
@@ -0,0 +1,1302 @@
+// C++ Headers
+#include <vector>
+#include <string>
+#include <list>
+#include <ostream>
+#include <algorithm>
+
+// ObjexxFCL Headers
+#include <ObjexxFCL/Array1D.hh>
+#include <ObjexxFCL/Array1S.hh>
+#include <ObjexxFCL/Array2D.hh>
+#include <ObjexxFCL/Array2S.hh>
+#include <ObjexxFCL/Array3D.hh>
+#include <ObjexxFCL/Optional.hh>
+
+// EnergyPlus Headers
+#include <OutputReportTabularAnnual.hh>
+#include <OutputReportTabular.hh>
+#include <InputProcessor.hh>
+#include <DataGlobals.hh>
+#include <OutputReportData.hh>
+#include <DataHVACGlobals.hh>
+#include <OutputProcessor.hh>
+#include <DataEnvironment.hh>
+#include <General.hh>
+#include <SQLiteProcedures.hh>
+#include <ScheduleManager.hh>
+
+
+namespace EnergyPlus {
+
+	namespace OutputReportTabularAnnual {
+
+		std::vector<AnnualTable> annualTables;
```

Comment:
@mbadams5 This may seem like a foolish question but how do I access the annualTables array from the unit test file that I am now trying to develop?  I am trying to test the GetInputTabularAnnual using EnergyPlusFixture.